tests/regression/74-invalid_deref/31-multithreaded.c

```diff
@@ -0,0 +1,21 @@
+//PARAM: --set ana.activated[+] useAfterFree --set ana.activated[+] threadJoins --set ana.path_sens[+] threadflag  --set ana.activated[+] memOutOfBounds --enable ana.int.interval --set ana.base.arrays.domain partitioned --set ana.base.privatization mutex-meet-tid
```

Comment:
Is all of this necessary? The program doesn't contain any arrays for example.

Also, what is being tested? That Goblint doesn't crash?

Reply:
That no warning is produced on line 18. I added the corresponding annotation.